.travis.yml

```diff
@@ -1,20 +1,45 @@
+sudo: required
+dist: trusty
 language: node_js
-sudo: false
+
 node_js:
-- '5.3'
-branches:
-  except:
-  - /^v[0-9\.]+/
+- '6.9.1'
+- 'stable'
+
+addons:
+  apt:
+    sources:
+    - ubuntu-toolchain-r-test
+    - google-chrome
+    packages:
+    - google-chrome-stable
+    - g++-4.8
```

Comment:
We want to run this against `firefox: 'latest'` as well:
```
addons:
  apt:
    sources:
    - ubuntu-toolchain-r-test
    - google-chrome
    packages:
    - google-chrome-stable
    - g++-4.8
  firefox: 'latest'
```

Note the indentation is important here.